### DISCUSSION TITLE
Signing in using a POST request crashes the backend server

### DISCUSSION THREAD

**Discussion by Adamo-O** [SOEN-341-Project/Group-P-soen341project2022#110] [Bugs] [answered]
# Background
After authentication was added to the backend, I saw this issue where sending a POST request to the 'api/users/signin' route will cause an error and crash the backend. 

# Replicating Issue
The 'signin' route is able to correctly return error response data when the user is not found, or the password is invalid.
The issue happens when the sign in details are **correct**. An ERROR status is returned and the image below shows the console output of the backend.
![image](https://user-images.githubusercontent.com/55419809/159782777-988a2a69-2375-4712-839e-6df384e80021.png)

# Assistance
I think @KooshaG would be able to help here.
Thanks!

**Comment by KooshaG** [comment, marked as answer]
Whoops, my bad. You need a new environment variable to run the backend locally. I posted the variable in the backend channel in the discord. 🐒

**Reply by Adamo-O** [reply]
Perfect, thanks!
